src/platforms/react-native/migration.mdx

```diff
@@ -4,6 +4,14 @@ sidebar_order: 100
 description: "Learn about migrating from 1.x to 2.x to enable release health tracking and native stack traces by default."
 ---
 
+## From 2.x to 3.x
+
+There are no changes needed when migrating from versions `2.x` to `3.x`, although you will need to make sure that you run `pod install` on iOS and update your native code on both platforms.
```

Comment:
if its iOS only, why do you say both platforms?

Reply:
Hmm guess this is bad wording, this is more of that you will need to rebuild your app after this update. Even though you're supposed to for every update, RN devs often use codepush which just updates the js code ota so this is just to make it clear that this will definitely need a rebuild.